symfony/framework-bundle/4.3/config/packages/framework.yaml

```diff
@@ -0,0 +1,24 @@
+framework:
+    secret: '%env(APP_SECRET)%'
+    #default_locale: en
+    #csrf_protection: true
+    #http_method_override: true
+
+    # Enables session support. Note that the session will ONLY be started if you read or write from it.
+    # Remove or comment this section to explicitly disable session support.
+    session:
+        handler_id: ~
+        cookie_secure: auto
+        cookie_samesite: lax
+
+    #esi: true
+    #fragments: true
+    php_errors:
+        log: true
+
+    validation:
+        # Enables validator auto-mapping support.
+        # For instance, basic validation constraints will be inferred from Doctrine's metadata.
+        auto_mapping:
+            App\Entity\: []
```

Comment:
I keep thinking that this might make more sense as an annotation o the class not here. Currently, I look at my class to find all my constraints. Now I will possibly have these “silent” constraints I can’t track down. Could we activate the auto-constraints... with a constraint? We could generate entities with this by default with make:entity.

I know, wrong place to discuss - I just didn’t make it in time to the original Pr.

Reply:
This is a config which applies to all entities in given namespace, not specific class. How would you do that with annotation? Where would you put such annotation?

Reply:
This config supports setting full namespaces, as here, but also a list of specific classes using the glob syntax.
There is no annotations because it’s a bundle config. Only XML and YAML are supported. It’s similar to the service discovery config.

Reply:
And in almost all cases, it’s safer to always enable autovalidation. If it triggers a false positive, it’s most likely because the user is doing something very wrong that should be fixed.